### PR TITLE
Download composer on docker build (efficiency)

### PR DESCRIPTION
**Pull request by imdadahad**
Refactor to download composer when building a docker container instead of downloading and installing each time.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -8,6 +8,7 @@ RUN \
 	&& apt-get install -y --no-install-recommends zlib1g-dev \
 		make \
   && docker-php-ext-install zip \
```
Comment (bandesz) [inline docker/Dockerfile]: fix the indentation here

File: docker/Dockerfile [hunk 2]
```diff
@@ -8,6 +8,7 @@ RUN \
 	&& apt-get install -y --no-install-recommends zlib1g-dev \
```
Comment (bandesz) [inline docker/Dockerfile]: the package name should be in the next line

File: docker/Dockerfile (hunk 1 above)
Comment (imdadahad) [inline docker/Dockerfile]: Fixed, thanks

File: docker/Dockerfile (hunk 2 above)
Comment (imdadahad) [inline docker/Dockerfile]: Changed.

**Comment by bandesz** [comment]
merge it! :)

**Pull request merged**